.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for your contribution. Please add a summary of the change and fill in the TYPE an DESC for automatic addition to HISTORY.md
+-->
+
+<!-- Automatic history file management:
+  Set the type to one of FORMAT, BREAKING_API, BREAKING_BEHAVIOR, FEATURE, IMPROVEMENT, DEPRECATION, BUG, C_API, CPP_API or NO_HISTORY.
+  NO_HISTORY should be used rarely. I it mostly associated with repo level changes (README/CI/etc) that do not need to be logged in
+  the HISTORY.md file for version updates.
+  Set the DESC to a one sentence or less summary of the change
```

Comment:
It's important to note that the DESC must be a single line. Maybe use "one line" instead of "one sentence"?